source/connect/advanced-connection.txt

```diff
@@ -12,3 +12,88 @@ Advanced Connection Tab
    :depth: 1
    :class: singlecol
 
+
+The :guilabel:`Advanced` connection tab provides additional connection options 
+for your deployment. These options can be used alongside the 
+:ref:`General <general-connection-tab>`, :ref:`Authentication 
+<authentication-connection-tab>`, :ref:`TLS / SSL <tls-ssl-tab>`, and 
+:ref:`Proxy / SSH Tunnel <ssh-connection>` connection options. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Advanced` tab.
+
+      Select a :guilabel:`Read Preference` from the following options:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50 
+
+         * - Read Preference
+           - Description
+
+         * - primary
```

Comment:
Consider making these preferences title case, because that's how they appear in the UI.